Change onGUI box for TwineNodes to wrap text and display all body text

grahamearley opened the pull request:
The GUI box will be 10px away from the top and left side of the screen. It's width is 1/3 of the screen width, and its height is the whole screen (so it will still not show all the text if the text goes beyond that length -- in the future we should modify this to change text size).

I also enabled word wrap in the box's style.

(Closes #57)

**Todo still:** We still have to deal with finalizing the layout of where this text goes, etc. For now, this mostly just makes the text readable for our demos.

grahamearley committed:
Change onGUI box for TwineNodes to wrap text and display all body text

grahamearley committed:
Extract frame width (since we refer to it twice)

cpimhoff commented:
I like this, but yeah, still todo is defining our screen real estate better. Like, how does this conflict with the GUI box of our (yet to be implemented) annotation summaries on mouse over?

cpimhoff reviewed (approved):
💪

grahamearley committed:
Create a copy of the Box GUI skin instead of editing the default skin

(editing the default ended up changing all the GUIs we displayed!
including the crosshair!)

grahamearley commented:
@cpimhoff agreed. Mostly doing this for demo purposes

brennan724 reviewed (approved):
Looks good!
